pyproject.toml

```diff
@@ -157,6 +160,9 @@ ignore = [
   "E731",
 ]
 
+[tool.ruff.mccabe]
```

Comment:
Ruff is saying this should be `[tool.ruff.lint.mccabe]`

I can just do this as part of the merge if you want.